### PR TITLE
refactors models sync with prod config (nginx)

### PR DESCRIPTION
**Pull request by pdyba**

### CODE REVIEW

File: frontend/src/components/learning/question.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<template>
+    <div class="panel panel-default">
+        <strong>{{ question.title }} </strong><span v-if="exercise.answared" class="badge badge-success">Done</span>
+        <br>
+        {{ question.task }}
+        <br>
+        <br>
+        <b-row>
+            <div class="editor_form">
+                <editor v-model="question.answare" @init="editorInit" lang="python" theme="chrome" width="100%"
```
Comment (tqZK) [inline frontend/src/components/learning/question.vue]: answare...

File: models/common.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    ]
+
+    async def create(self):
+        self.password = hash_string(self.password)
+        return await super().create()
+
+    @staticmethod
+    async def validate_password(new_password):
+        if len(new_password) < 8:
+            return {"success": False, "msg": "You provided too short password"}
```
Comment (tqZK) [inline models/common.py]: Password needs to be at least 8 characters long

File: models/learning.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        Column('id', Integer, primary_key=True),
+        Column('title', String(255)),
+        Column('description', String(10000)),
+        Column('users', ForeignKey('users'), default=DEFAULT_USER),
+        Column('time_created', DateTime(), default=datetime.utcnow),
+    ]
+
+    async def get_question(self, question_order=1):
+        if question_order + 1 >= await self.get_question_amount():
+            return {'last': True, 'msg': 'That was the last question in the quiz.'}
```
Comment (tqZK) [inline models/learning.py]: 'That was the last question of the quiz.' ? tu i w innych miejscach

File: nginx_nocertbot.conf [hunk 4]
```diff
@@ -0,0 +1,7 @@
+server {
+  listen 80;
+  server_name pylove.pl www.pylove.pl pylove.org www.pylove.org;
```
Comment (tqZK) [inline nginx_nocertbot.conf]: pylove.pl?

Comment (pdyba) [reply nginx_nocertbot.conf]: mamy tez i ta domene ;)

**Pull request merged**